pkgs/applications/version-management/subversion/default.nix

```diff
@@ -64,10 +60,9 @@ let
           "man"
         ];
 
-        nativeBuildInputs = lib.optionals needsAutogen [
+        nativeBuildInputs = [
           autoconf
           libtool
-          python3
```

Comment:
I'm seeing a subversion configure error for all platforms about missing python in PATH for autogen.sh

Reply:
Same as ofborg reports:

```bash
  > Copying autoconf helper: /nix/store/3xazlg7rj74565l14msbqlihhw50v24j-libtool-2.4.7/share/libtool/build-aux/config.sub
       > Python 2.7 or later is required to run autogen.sh
       > If you have a suitable Python installed, but not on the
       > PATH, set the environment variable PYTHON to the full path
       > to the Python executable, and re-run autogen.sh
       For full logs, run 'nix log /nix/store/a6yja4533vpg83s0qr7wiwakln2m6w7m-subversion-1.14.5.drv'.

Reply:
I swaer this was building for me before. Will look into it later.

Reply:
Should be fixed now.